### ISSUE TITLE
Reloading a file that was edited in another editor does not work.

### ISSUE THREAD

**Issue by ranyadshalom** [lwblackledge/file-watcher#14]
I am using Atom 1.10.2 & editing my file in PyCharm.

When I change the file in PyCharm, I get a prompt that the file was changed, but when I hit  "Reload", the file just closes, and does not reload.

I think it has to do with Atom messing with the file too: whenever a change was made to it by another process, Atom automatically loads the old version as 'filename.ext__jb_old__' (and closes the original). So the dialog I get from file-watcher actually refers to 'filename.ext__jb_old__' as the file that was changed!

**TL; DR:**  I believe a change in the latest Atom version is causing that problem but since I just started using it  I do not know for sure.

**Comment by lwblackledge** [comment]
I'll download PyCharm and try to reproduce this.

**Comment by lwblackledge** [comment]
Okay from what I see this is PyCharm's doing. On save, it actually renames the current file to `{filename}___jb_old___`, saves as the new file name, then deletes the `___jb_old___` file. It does this even when file-watcher is disabled (you can see it if you make changes to a file in Atom but don't save, then modify the file in PyCharm)

Looking at the PyCharm docs you can actually turn this behaviour off:

### [Safe Write](https://www.jetbrains.com/help/webstorm/2016.3/system-settings.html)
> If this check box is selected, a changed file is first saved in a temporary file. If the save operation succeeds, the file being saved is replaced with the saved file. (Technically, the original file is deleted and the temporary file is renamed.) 
> Also, the ownership of such file changes.

![safewrite](https://cloud.githubusercontent.com/assets/1142220/22176290/ddadf51e-dfd4-11e6-8cae-6439e936451d.png)

As such I don't think this is something that I will fix since it is specific to PyCharm (and possibly other Jetbrains IDEs) but they already offer a workaround.
